src/hotspot/share/gc/g1/g1OldGenAllocationTracker.cpp

```diff
@@ -50,7 +50,7 @@ void G1OldGenAllocationTracker::reset_after_gc(size_t humongous_bytes_after_gc)
   _humongous_bytes_after_last_gc = humongous_bytes_after_gc;
 
   log_debug(gc, alloc, stats)("Old generation allocation in the last mutator period, "
-                              "old gen allocated: " SIZE_FORMAT "B, humongous allocated: " SIZE_FORMAT "B,"
+                              "old gen allocated: " SIZE_FORMAT "B, humongous allocated: " SIZE_FORMAT "B, "
```

Comment:
You could replace uses of `SIZE_FORMAT` with "%zu" formatting directives, which might allow removal of the line break in the string without making it overly long.